assignments/prototypes.js

```diff
@@ -55,22 +86,122 @@
   - A crashed car can't be driven any more. Attempts return a string "I crashed at x miles!", x being the miles in the odometer.
   - Give cars the ability to be repaired.
   - A repaired car can be driven again.
+  */
+function Car (Model_Name, Make){
+  this.model = Model_Name;
+  this.make = Make;
+  this.driveDistance = true;
+  this.crashAble = true;
+  this.canRepair = true;
+}
+
+let odometer = 0;
+Car.prototype.drvienFar = function(){
+  if(this.driveDistance=== true){
+    return odometer = odometer + 5;
+  }else{
+    return  `I crashed at ${odometer} miles!.`
+
+  }
```

Comment:
There's a better way of doing this, you could  pass distance in as a function and add it to odometer